src/components/HeaderButton/style.ts

```diff
@@ -19,7 +19,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     },
 
     '&:hover': {
-      background: colors.invariant.componentOut3
+      background: '#4B5983'
```

Comment:
we haven't got this color in our design system?

Reply:
background added to themes

Reply:
done